dashboard/db/migrate/20171117010623_add_share_with_regional_partners_to_census_submissions.rb

```diff
@@ -0,0 +1,5 @@
+class AddShareWithRegionalPartnersToCensusSubmissions < ActiveRecord::Migration[5.0]
+  def change
+    add_column :census_submissions, :share_with_regional_partners, :boolean
+  end
+end
```

Comment:
Please split the migration (and associated schema and annotations) up from the rest of the PR. 

We follow this practice for existing tables to make rollback and revert easier in case of a problem.

Reply:
Done: #19291